Source/WebCore/Modules/WebGPU/GPUSupportedLimits.cpp

```diff
@@ -53,6 +53,11 @@ uint32_t GPUSupportedLimits::maxBindGroups() const
     return m_backing->maxBindGroups();
 }
 
+uint32_t GPUSupportedLimits::maxBindingsPerBindGroup() const
+{
+    return m_backing->maxBindingsPerBindGroup();
```

Comment:
Usually this data flow is backwards; information doesn't usually flow from the backing to the front-ing. Are you sure you want to ask the backing, instead of just having the information be owned right here in `GPUSupportedLimits` and set at creation time?

Reply:
I think the change is outside the scope of this PR however isn't this the pattern we follow with other GPU* classes?

For instance, every ::label implementation looks like:
```
String GPU[Buffer]::label() const
{
    return m_backing->label();
}
```

We can certainly change it and I think having the information stored directly in GPUSupportedLimits as suggested would be better.